Client/Info.plist

```diff
@@ -154,5 +156,10 @@
 	</array>
 	<key>UIFileSharingEnabled</key>
 	<true/>
+        <key>com.apple.developer.nfc.readersession.iso7816.select-identifiers</key>
+        <array>
+    	    <string>A000000527471117</string>
```

Comment:
What's this magic numbers?

Reply:
This is the tag's AID (Application Identifier). A complete list can be found here (even contains Yubico's other AIDs): https://www.eftlab.com/knowledge-base/211-emv-aid-rid-pix/

but that specific AID (`A000000527471117`) doesn't seem to be on the list. I assume it's new and possibly belonging to a beta key or a new Yubico key or something?

Reply:
You can find more details about the identifiers here: https://github.com/Yubico/yubikit-ios/blob/master/docs/NFC-Notes.md